cli/foundation-models/system/inference/image-text-embeddings/deploy-batch.yaml

```diff
@@ -0,0 +1,9 @@
+$schema: https://azuremlschemas.azureedge.net/latest/batchDeployment.schema.json
+name: demo
+description: "Batch endpoint for image and text embeddings task"
+type: model
+resources:
+    instance_count: 1
+settings:
+    mini_batch_size: 1
```

Comment:
Forgetting what we discussed previously: why not set the batch size higher?

Reply:
It seems we generally use batch size 1 for our examples. It could be a relatively arbitrary number as long as the number of samples in the csv files changes accordingly.